content/installation/python/PythonInstall.md

```diff
@@ -101,6 +101,16 @@ app = get_wsgi_application()
 app = ContrastMiddleware(app)
 ```
 
+#### Pylons
+
+``` python
+from pylons.wsgiapp import PylonsApp
+from contrast.agent.middlewares import WSGIMiddleware
```

Comment:
This should be `from contrast.agent.middlewares.wsgi_middleware import WSGIMiddleware`. For consistency I also think we should add `as ContrastMiddleware` (a few lines down too)

Reply:
Let me update that!

Reply:
udpated

Reply:
spelling is hard